config/test/conf/schema.xml

```diff
@@ -136,7 +136,7 @@
    <field name="Language" type="string" indexed="true" stored="true" required="false" multiValued="false" termVectors="false" termPositions="false" termOffsets="false"/>
    <field name="ModificationDate" type="date" indexed="false" stored="true" required="false" multiValued="false" termVectors="false" termPositions="false" termOffsets="false"/>
 <!-- catchall field, containing all other searchable text fields (implemented via copyField further on in this schema -->
-   <field name="SearchableText" type="text_it" indexed="true" stored="false" required="false" multiValued="false" termVectors="false" termPositions="false" termOffsets="false"/>
+   <field name="SearchableText" type="text_it" indexed="true" stored="true" required="false" multiValued="true" termVectors="false" termPositions="false" termOffsets="false"/>
```

Comment:
@mamico is it ok for you? It seems that tika sets SearchableText as a list of values

Reply:
multivalued ok. why stored?

Reply:
only for debug (i copied it from dev configuration)